src/sentry/api/endpoints/group_ai_summary.py

```diff
@@ -114,13 +114,14 @@ def _call_seer(
             option=orjson.OPT_NON_STR_KEYS,
         )
 
+        url, salt = get_seer_salted_url(f"{settings.SEER_AUTOFIX_URL}{path}")
```

Comment:
minor nit - should we call this `salted_url` instead of salt?

Reply:
In the old code path, the salt was the url, but in the new code path, the salt is just the nonce itself, minus url context.  So that's why I kept the name more ambiguous.  It does return a url, which is the url with the salt (nonce) added separately from the signature.

Reply:
Actually my bad. It should be just salt.